lib/tinyset.rs

```diff
@@ -65,13 +79,18 @@ impl<T: fmt::Debug + u64set::Fits64 + Ord> fmt::Debug for tinyset_Set64<T> {
 
 impl<T: u64set::Fits64 + Ord> Ord for tinyset_Set64<T> {
     fn cmp(&self, other: &Self) -> cmp::Ordering {
-        self.x.iter().cmp(other.x.iter())
+        let size_cmp = self.x.len().cmp(&other.x.len());
+        if size_cmp != cmp::Ordering::Equal {
+            size_cmp
+        } else {
+            self.iter().cmp(other.iter())
+        }
     }
 }
 
 impl<T: u64set::Fits64 + Ord> PartialOrd for tinyset_Set64<T> {
     fn partial_cmp(&self, other: &Self) -> Option<cmp::Ordering> {
-        self.x.iter().partial_cmp(other.x.iter())
+        Some(self.cmp(other))
```

Comment:
How about returning `None` for sets of different size?

Reply:
My `Ord` implementation actually says that a smaller-size set is also smaller in the total order (which makes a big difference for performance), so `PartialOrd` just inherits this semantics.